.gitmodules

```diff
@@ -70,6 +70,9 @@
 [submodule "third_party/capstone"]
 	path = third_party/capstone
 	url = https://github.com/capstone-engine/capstone
+[submodule "third_party/open-bp-cpp/third_party/IXWebSocket"]
```

Comment:
OPEN BP CPP IS HEAVILY MODIFIED AS I MADE IT GOOD WHILE IT WAS HORRIBLE BEFORE

Reply:
I am sorry but I am not going to go through 20 thousand lines of code to make sure you didn't add a backdoor (heh get it?) somewhere in the program.

Reply:
I dun geddit

Reply:
Well you see, prior this year we had some "guys" that tried to implement a backdoor in the OSS project named XZ. Here, OFFTKP wants to make sure that the program stays secure. In order to do that, every PR that adds code must be analyzed by hand to prevent any attack of this kind. Reviewing code is an operation that take an exponential amount of time. The more line there is to review, the more time it takes. I totally understand that OFFTKP does not have the required time to analyze 20K lines of code. Only peoples employed in an enterprise would do this. If he/she is a hobbyist or a developer doing this on his/her free time, then he/she may not have this time.

Reply:
> Well you see, prior this year we had some "guys" that tried to implement a backdoor in the OSS project named XZ. Here, OFFTKP wants to make sure that the program stays secure. In order to do that, every PR that adds code must be analyzed by hand to prevent any attack of this kind. Reviewing code is an operation that take an exponential amount of time. The more line there is to review, the more time it takes. I totally understand that OFFTKP does not have the required time to analyze 20K lines of code. Only peoples employed in an enterprise would do this. If he/she is a hobbyist or a developer doing this on his/her free time, then he/she may not have this time.  

1. https://en.m.wikipedia.org/wiki/Singular_they

2. I agree, either they (the author of this PR) rewrite to use the "official" branch of the library or they try to make a PR to make their branch of the library merged to the library itself.

Reply:
>     1. https://en.m.wikipedia.org/wiki/Singular_they

Thank you, I am not en English people so I didn't know it.

Reply:
> Well you see, prior this year we had some "guys" that tried to implement a backdoor in the OSS project named XZ. Here, OFFTKP wants to make sure that the program stays secure. In order to do that, every PR that adds code must be analyzed by hand to prevent any attack of this kind. Reviewing code is an operation that take an exponential amount of time. The more line there is to review, the more time it takes. I totally understand that OFFTKP does not have the required time to analyze 20K lines of code. Only peoples employed in an enterprise would do this. If he/she is a hobbyist or a developer doing this on his/her free time, then he/she may not have this time.

A true 10X programmer like @OFFTKP would have no problem with 20K lines of code. A real 69X programmer like me would just look at the diff between upstream and my vendored version in a text editor 🤣